### ISSUE TITLE
Importing tool

### ISSUE THREAD

**Issue by fappaizuri** [biotorrents/gazelle#100]
Can you create a tool to import torrentrader into biogazelle? I am interested to import my tracker into this nice source.

**Comment by pjc09h** [comment]
Thanks for your kind words. Is this the correct repo? https://github.com/Arkhana/TorrentTrader-3.0

Looking at the database schema, I'd say it's highly unlikely. I'd also advise you that BioGazelle is nowhere near production ready. While most of the core object code is worked out, the underlying database is a mess, so I effectively have to create a tool to import legacy What.cd Gazelle into BioGazelle.

**Comment by fappaizuri** [comment]
Yes, it is the correct repo of torrenttrader. So it will be a pain to import the database into BioGazelle.
